### PR TITLE
Error & info messages, code comments language fixes part 1

### PR DESCRIPTION
**Pull request by llewellyn-sl**

### CODE REVIEW

**Review by abhi18av — APPROVED** [review]
Looking good @llewellyn-sl , are you planning to continue working on this PR w.r.t the entire codebase or this is ready to be merged?

**Comment by llewellyn-sl** [comment]
> Looking good @llewellyn-sl , are you planning to continue working on this PR w.r.t the entire codebase or this is ready to be merged?

@abhi18av thanks for reviewing. I'm working my way through the entire codebase, but this is going to be stretched across the next few weeks. I think it would be best to do this in chapters with separate PRs each time.

File: modules/nextflow/src/main/groovy/nextflow/NextflowMeta.groovy [hunk 1]
```diff
@@ -23,7 +23,7 @@ import static nextflow.extension.Bolts.DATETIME_FORMAT
 @EqualsAndHashCode
 class NextflowMeta {
 
-    private static final String DSL1_EOL_MESSAGE = "Nextflow DSL1 is no longer supported — Please update your script to DSL2 or use Nextflow 22.10.x or an earlier version"
+    private static final String DSL1_EOL_MESSAGE = "Nextflow DSL1 is no longer supported. Update your script to DSL2, or use Nextflow 22.10.x or earlier."
```
Comment (pditommaso) [inline modules/nextflow/src/main/groovy/nextflow/NextflowMeta.groovy:26]:
```suggestion
    private static final String DSL1_EOL_MESSAGE = "Nextflow DSL1 is no longer supported — Update your script to DSL2, or use Nextflow 22.10.x or earlier"
```

We usually don't use dot in error messages

File: modules/nextflow/src/main/groovy/nextflow/file/DirWatcherV2.groovy [hunk 2]
```diff
@@ -114,7 +114,7 @@ class DirWatcherV2 implements DirListener, FileAlterationListener {
     void apply( Closure onNext ) {
         this.onNext = onNext
         if( !base.isDirectory() ) {
-            log.warn "Cannot watch a non-existent directory: $base -- Make sure that the path exists and is a directory"
+            log.warn "Cannot watch a non-existent directory: $base — Ensure the path is a valid directory"
```
Comment (pditommaso) [inline modules/nextflow/src/main/groovy/nextflow/file/DirWatcherV2.groovy:117]:
```suggestion
            log.warn "Cannot watch a non-existent directory: $base -- Ensure the path is a valid directory"
```

File: modules/nextflow/src/main/groovy/nextflow/NextflowMeta.groovy [hunk 3]
```diff
@@ -174,7 +174,7 @@ class NextflowMeta {
             return ver
         }
         else if( mode == 'preview' )
-            throw new IllegalArgumentException("Preview nextflow mode 'preview' is not supported anymore -- Please use `nextflow.enable.dsl=2` instead")
+            throw new IllegalArgumentException("Preview nextflow mode ('preview') is no longer supported — use `nextflow.enable.dsl=2` instead")
```
Comment (pditommaso) [inline modules/nextflow/src/main/groovy/nextflow/NextflowMeta.groovy:177]:
```suggestion
            throw new IllegalArgumentException("Preview nextflow mode ('preview') is no longer supported —- use `nextflow.enable.dsl=2` instead")
```

**Review by pditommaso — APPROVED** [review]
Nice, thanks

File: modules/nextflow/src/main/groovy/nextflow/NextflowMeta.groovy (hunk 1 above)
Comment (pditommaso) [reply modules/nextflow/src/main/groovy/nextflow/NextflowMeta.groovy]: welcome!

**Pull request merged**